dbfit-java/teradata/.gitignore

```diff
@@ -0,0 +1 @@
+/bin/
```

Comment:
> That's my bad (wild card used in git add).

I think there's a way to configure your own ignore settings (in `.git/info/exclude or something)